crates/ruff_python_parser/src/snapshots/ruff_python_parser__lexer__tests__fstring_with_format_spec.snap

```diff
@@ -156,8 +183,11 @@ expression: lex_source(source)
     (
         FStringMiddle {
             value: "*^",
-            is_raw: false,
-            triple_quoted: false,
+            flags: StringFlags(
+                StringFlagsInner(
+                    DOUBLE | F_PREFIX,
+                ),
```

Comment:
Nit: It might be worth implementing `Debug` for `StringFlags` to get a nicer representation (I think that should be straightforward after introducing `StringPrefix` because you can than do:

```rust
f.debug_struct("StringFlags").field("prefix", &self.prefix()).field("quote_style", &self.quote_style()).finish()
```